rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/ProductMilestoneEndpoint.java

```diff
@@ -216,6 +219,28 @@ ProductMilestone patchSpecific(
     @Path("/{id}/close")
     void cancelMilestoneClose(@Parameter(description = PM_ID) @PathParam("id") String id);
 
+    @Operation(
+            summary = "Get milestone releases.",
+            responses = {
+                    @ApiResponse(
+                            responseCode = SUCCESS_CODE,
+                            description = SUCCESS_DESCRIPTION,
+                            content = @Content(schema = @Schema(implementation = BuildPage.class))),
+                    @ApiResponse(
+                            responseCode = INVALID_CODE,
+                            description = INVALID_DESCRIPTION,
+                            content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
+                    @ApiResponse(
+                            responseCode = SERVER_ERROR_CODE,
+                            description = SERVER_ERROR_DESCRIPTION,
+                            content = @Content(schema = @Schema(implementation = ErrorResponse.class))) })
+    @GET
+    @Path("/{id}/releases")
+    Page<ProductMilestoneRelease> getMilestoneReleases(
```

Comment:
I don't really like the Release name in it. It can be confused with ProductRelease